src/o365/teams/commands/report/teams-report-useractivityuserdetail.ts

```diff
@@ -49,8 +55,39 @@ class TeamsReportUserActivityUserDetailCommand extends GraphCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        cmd.log(res);
-
+        if (args.options.outputFile) {
+
+          let content: string = '';
+          
+          if (args.options.outputFile.toLowerCase().endsWith('.json'))
```

Comment:
You should check if the `--output` option is set to json instead of checking for extension.

Reply:
I have fixed this in the latest commit. Just an FYI here user can export the report data to the specified path in csv,txt and json formats